Xamarin.Forms.Platform.UAP/WindowsBasePlatformServices.cs

```diff
@@ -30,11 +30,15 @@ internal abstract class WindowsBasePlatformServices : IPlatformServices
 		const string WrongThreadError = "RPC_E_WRONG_THREAD";
 		readonly CoreDispatcher _dispatcher;
 		readonly UISettings _uiSettings = new UISettings();
+		private readonly bool _isThreadingSupported = true;
```

Comment:
This should be a static field, as it won't change during the course of the app.

Reply:
that's true, but it seems like the `WindowsBasePlatformServices` type is only created once anyway, when the app is launched.

Reply:
Right!